typings/index.d.ts

```diff
@@ -448,10 +448,10 @@ export class CommandInteractionOptionResolver {
   public get(name: string, required: true): CommandInteractionOption;
   public get(name: string, required?: boolean): CommandInteractionOption | null;
 
-  public getSubCommand(required?: true): string;
-  public getSubCommand(required: boolean): string | null;
-  public getSubCommandGroup(required?: true): string;
-  public getSubCommandGroup(required: boolean): string | null;
+  public getSubcommand(required?: true): string;
```

Comment:
You missed on instance:
https://github.com/discordjs/discord.js/blob/4aa367f891e9e66384dc96636848e359f2c60ee4/typings/index.d.ts#L434